intranet/static/js/eighth/statistics.js

```diff
@@ -72,4 +75,4 @@ $(document).ready(function() {
             $(this).append("<td><div class='sparkline' style='width:" + Math.round(100 * val / max) + "px'></div></td>");
         });
     }
-});
+});
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Newline required at end of file but not found.](https://www.codacy.com/app/tjcsl/ion/file/2769584656/issues/source?bid=3176160&fileBranchId=3355598#l78)